src/setup_utilities/types.ts

```diff
@@ -43,7 +43,6 @@ export type ToolChainDictionary = { [name: string]: ToolChainEntry };
 
 export interface GlobalConfig {
   toolchains: ToolChainDictionary,
```

Comment:
Is this toolchains variable use in the code base, otherwise it can be removed and added to the deprecated list